src/hooks/useViewportSize/useViewportSize.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 
-export interface ViewportSize {
+export interface ViewportSizeResult {
```

Comment:
We should keep this type as is, and create `UseViewportSizeResult` type as an alias to this type

Reply:
Up to this point, we haven't exported this hook yet, so no one can use it. That's why I renamed without an alias. Are you thinking of making an alias after all?

Reply:
Then `Use` should be at the beginning

Reply:
done